index.html

```diff
@@ -38,29 +38,29 @@ <h2>Participants</h2>
       Check out the amazing sites by participants below!
 
       <div class="sites participants" id="sites">
-        <a href="participants/kat-goff/">Kat Goff</a>
-        <a href="participants/verity-brean/">Verity Brean</a>
```

Comment:
It looks like Verity was removed but not added back in, please re-add.

Reply:
Along with `_blank` so that the link will open in a new tab: `<a href="participants/verity-brean/">Verity Brean</a>`

Reply:
> It looks like Verity was removed as a contributor but not added back in, please re-add 🙂

I have fixed it.

Reply:
Excellent, thank you!